dcos/dcos-ui
# DCOS-51824: In Service Create/Edit form, expand the advanced settings if they contain errors

Expand the advanced settings if they contain error
so that the user can see the errors.

Closes https://jira.mesosphere.com/browse/DCOS-51824

## Testing
1. In Services tab, edit or create a single-container service.
2. In the first tab, click "More Settings" and add some error there, for example set Disk to "-1".
3. Close "More Settings".
4. Try to run the service.
5. Verify that "More Settings" was expanded.
6. Fix the error and close "More Settings".
7. Run the service (with another error, for example missing id) and verify that "More Settings" wasn't opened.

## Trade-offs
While working on this, I discovered a bug that also happens on master: https://jira.mesosphere.com/browse/DCOS-52162 .

## Dependencies
None.

## Screenshots
### Before
![Peek 2019-04-15 16-34](https://user-images.githubusercontent.com/40791275/56137515-b6081d00-5f9d-11e9-8872-3d227253eaa7.gif)

### After
![Peek 2019-04-15 16-44](https://user-images.githubusercontent.com/40791275/56137597-ddf78080-5f9d-11e9-8d7b-a08a92bfa00c.gif)
